src/test/java/rv/atqc/gttesting/archex/LogInPage.java

```diff
@@ -1,11 +1,52 @@
 package rv.atqc.gttesting.archex;
 
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import rv.atqc.gttesting.res.Resources;
 
 public class LogInPage extends LeftNavigationForGuestUserPage<LogInPage> {
 
-	public LogInPage(WebDriver driver) {
-		super(driver);
-	}
+    @FindBy(css = "html.ng-scope body header nav.navbar.navbar-default.navbar-fixed-top.navbar-style.ng-scope div.container.nav-padding div#navbar.collapse.navbar-collapse ul.nav.navbar-nav li.dropdown.open ul.dropdown-menu.dropdown-auth.nav-ul auth.ng-isolate-scope div.ng-scope div.container.registration.ng-scope form.ng-dirty.ng-valid-parse.ng-valid-required.ng-valid-email.ng-valid.ng-valid-pattern input.btn.btn-success.btn-block.submit-btn")
+    private WebElement loginButton;
+    @FindBy(css = "html.ng-scope body header nav.navbar.navbar-default.navbar-fixed-top.navbar-style.ng-scope div.container.nav-padding div#navbar.collapse.navbar-collapse ul.nav.navbar-nav li.dropdown a.dropdown-toggle i.fa.fa-user.navtop")
+    private WebElement leftTopGreenButton;
```

Comment:
It should not be here!! For operate this you must create new HeadForGuestPage().clickLeftNavigationButton(). It is web element from HeadPage